Explorer/Assets/Scripts/ECS/StreamableLoading/AssetBundles/LoadAssetBundleSystem.cs

```diff
@@ -147,8 +151,14 @@ private async UniTask<GameObject> LoadAllAssetsAsync(AssetBundle assetBundle, Ca
 
                 if (!depResult.Succeeded)
                     throw new Exception($"Dependency {hash} resolution failed", depResult.Exception);
+
+                return depResult.Asset;
+            }
+            catch (OperationCanceledException)
+            {
+                assetBundlePromise.ForgetLoading(World);
+                return null;
```

Comment:
I think we should re-throw instead so `OperationCancelledEzxception` can be handled in `LoadSystemBase`